### ISSUE TITLE
What is the monitor of lr_scheduler?

### ISSUE THREAD

**Issue by Co1lin** [PyTorchLightning/pytorch-lightning#8552]
### Discussed in https://github.com/PyTorchLightning/pytorch-lightning/discussions/8541

<div type='discussions-op-text'>

<sup>Originally posted by **Co1lin** July 24, 2021</sup>
Hi! I’m wondering what is `metric_to_track` for the parameter monitor  of a learning rate scheduler? I see the example that uses `"val_loss"` as its value, but I don’t know is `val_loss` a name of a variable (where should I declare this variable?) or a keyword that can be recognized by pytorch_lightning such that it will monitor the loss of the `validation_step`?</div>

I get a warning saying:

```shell
packages/pytorch_lightning/callbacks/model_checkpoint.py:610: LightningDeprecationWarning: Relying on `self.log('val_loss', ...)` to set the ModelCheckpoint monitor is deprecated in v1.2 and will be removed in v1.4. Please, create your own `mc = ModelCheckpoint(monitor='your_monitor')` and use it as `Trainer(callbacks=[mc])`.
```
I'm not sure whether this usage will be deprecated for scheduler's monitor too.

**Comment by ethanwharris** [comment]
Hi @Co1lin, you can configure the monitor for LR scheduling in the `configure_optimizers` hook of your `LightningModule` as detailed here: https://pytorch-lightning.readthedocs.io/en/stable/common/optimizers.html?highlight=lr%20scheduler#learning-rate-scheduling

Hope that helps :smiley:
